app/service/planning_svc.py

```diff
@@ -97,7 +98,7 @@ async def get_links(self, operation, buckets=None, agent=None, trim=True, planne
                 t = []
                 for bucket in buckets:
                     t.append([ab for ab in abilities for b in ab.buckets if b == bucket])
-                abilities = t
+                abilities = [ability for ab in t for ability in ab]
```

Comment:
Okay my stupid brain is not fully processing this... is this impacted by using the index versus the ability_id? I don't see the link and it looks like we are just flattening out `t`?

Reply:
It's literally just flattening out t - there's some weird behavior further on if you have a list of lists at this point, so the cleanest option was just to flatten it. This is an unrelated bug patch.